packages/app/src/runner/event-manager.ts

```diff
@@ -449,10 +449,20 @@ export class EventManager {
     this.studioStore.setup(config)
 
     const isDefaultProtocolEnabled = Cypress.config('isDefaultProtocolEnabled')
-    const isStudioProtocolEnabled = Cypress.config('isStudioProtocolEnabled')
```

Comment:
Let's make sure we remove all the references to this config object (do a code search for it).